### DISCUSSION TITLE
Incongruent Response Curves and Contribution Calculations in terms of ROI

### DISCUSSION THREAD

**Discussion by xijianlim** [google/lightweight_mmm#49] [General]
Hi all, this is something I've been noticing the provided code base using the adstock-hill . The response curves (in terms of ROI) do not match the contribution/cost ratios in the contribution dataframe outputs. This is one example:
![image](https://user-images.githubusercontent.com/25444256/182072679-46c57b9f-de0f-48cd-a47b-fee1602880cf.png)

As you can see, the ROI via the function `plot.create_media_baseline_contribution_df` will provide a ROI 11.9 (Contribution from the training set is 2820, impressions is 9468 and an average price of 0.025 would mean $236 for cost, yielding $11.9).

However, the Response curves clearly show a ROI well below 1 and hitting its diminishing profile.

To help emphasize this discrepancy, ive amended the code to return a dataframe from "plot.plot_response_curves" 
![image](https://user-images.githubusercontent.com/25444256/182073508-fe216383-ddd8-4020-8128-e335496a10ba.png)

This example was found using the util functions to generate the data.

**Comment by pabloduque0** [comment]
Hello @xijianlim ! I will answer on the issue tab instead of here.
